### PR TITLE
Added logo

### PR DESCRIPTION
**Pull request by Meggie-H**

### CODE REVIEW

File: src/assets/logos/ttt-logo.png
Comment (tanadendere) [inline src/assets/logos/ttt-logo.png:1]: 🔥🤩

**Review by tanadendere — APPROVED** [review]
The logo is sooo nice!!

**Review by AyaazStreet — CHANGES_REQUESTED** [review]
I just sent you the square version for favicon.

**Review by AyaazStreet — APPROVED** [review]
I added the square icons

**Review by LesediMoko — APPROVED** [review]
Looks good! Nice work 👏

**Pull request merged**